openassessment/xblock/peer_assessment_mixin.py

```diff
@@ -161,12 +166,13 @@ def peer_path_and_context(self, continue_grading):
         path = 'openassessmentblock/peer/oa_peer_unavailable.html'
         finished = False
         problem_closed, reason, start_date, due_date = self.is_closed(step="peer-assessment")
-        user_service = self.runtime.service(self, 'user')
+        user_preferences = get_user_preferences(self.runtime.service(self, 'user'))
 
```

Comment:
Refactored the user timezone service to also retrieve the user_language (needed for time localization).